src/app/util/ember-compatibility-functions.cpp

```diff
@@ -248,6 +248,34 @@ bool ServerClusterCommandExists(const ConcreteCommandPath & aCommandPath)
 
 namespace {
 
+CHIP_ERROR ReadClusterDataVersion(const EndpointId & aEndpointId, const ClusterId & aClusterId, DataVersion & aDataVersion)
+{
+    EmberAfCluster * cluster = emberAfFindCluster(aEndpointId, aClusterId, CLUSTER_MASK_SERVER);
+    if (cluster == nullptr)
+    {
+        return CHIP_ERROR_NOT_FOUND;
+    }
+    aDataVersion = *(cluster->version);
```

Comment:
Please add an accessor on EmberAfCluster() instead of making assumptions about how to get from one to the value.

Reply:
Done, thanks